cookbook/ifan02.rst

```diff
@@ -0,0 +1,181 @@
+Ifan02
+======
+
+.. seo::
+    :description: Instructions for using Sonoff ifan02 in ESPHome.
+    :keywords: Fan, Sonoff, ifan02
+
+
+Configuration for `Sonoff ifan02 <https://www.itead.cc/sonoff-ifan02-wifi-smart-ceiling-fan-with-light.html>`__.
+
+First you need a :doc:`components/output/custom.html` to control the ifan02.
+
+Create a ifan02.h file:
+
+.. code-block:: c++
+
+    #include "esphome.h"
+    using namespace esphome;
+
+    class IFan02Output : public Component, public output::FloatOutput {
```

Comment:
```suggestion
    class IFan02Output : public output::FloatOutput {
```

this component has no setup/loop function, so it doesn't need any component functionality.